pkg/cmd/grafana-server/main.go

```diff
@@ -169,6 +176,31 @@ func executeServer(configFile, homePath, pidFile, packaging string, traceDiagnos
 	return nil
 }
 
+// registerServices registers Google Wire instantiated services with the dependency graph.
+// TODO: Remove after migration to Wire is completed.
```

Comment:
what do you remove? is it that this can move to wire generated code?

Reply:
@torkelo `registerServices` should be removed after migration to Wire is complete, since then the dependency graph is no longer necessary (it's then completely made by Wire).